### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by mertaliaslan31** [QWERTxD/BetterDiscordPlugins#174]
Assign Badges plugin works, but we cannot assign badges on your profile and other users.

**Comment by IamNickerson** [comment]
Yeah we cant Assign Badges since the text Assign Badges isn't showing up but badge is still appearing in the User.
The Badge is still appearing on the User:
![image](https://user-images.githubusercontent.com/61643704/152544581-205b8e1e-fb78-4f6e-8eed-052a2d9c2de9.png)
